src/client/components/app/Transactions/EnableSpigotTx.tsx

```diff
@@ -200,25 +237,64 @@ export const EnableSpigotTx: FC<EnableSpigotTxProps> = (props) => {
     );
   }
 
+  const renderRevFunc = () => {
+    if (revContractABI) {
+      return (
+        <TxFuncSelector
+          headerText="2. Rev Contract Funcs"
```

Comment:
This needs to be hooked up to our     public/locales/en. All user facing text needs to go through transaltion set up